packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -18,6 +18,7 @@ There are many possibilities for policy use:
 * Change the returned scopes of the access token and add claims to it and to the ID token using [inline hooks](/docs/concepts/inline-hooks/).
 * Notify other systems in real-time through an API when a user signs in.
 * Use the scopes of a token to look up user information in an external database or API, then add that data to the user's profile object.
+* Monitor your org for identity threats like user risk-level changes and changes in session context.
```

Comment:
```suggestion
* Monitor your org for identity threats like changes in entity risk or session context.
```

Reply:
Barbara's editorial guidance was for us to change references to `user risk levels` to `entity risk`